functions/join.go

```diff
@@ -335,6 +335,7 @@ func JoinNetwork(flags *viper.Viper) (*config.Node, *config.Server, error) {
 	}
 	logger.Log(1, "network:", node.Network, "node created on remote server...updating configs")
 	config.UpdateServerConfig(&joinResponse.ServerConfig)
+	config.UpdateHost(&joinResponse.Host)
```

Comment:
will also need to call this in register

Reply:
registerResponse does not include host data so this call cannot be made from register function